pydoctor/sphinx_ext/help_output.py

```diff
@@ -0,0 +1,38 @@
+"""
+Produce the pydoctor help output to be included in the documentation.
+"""
+from docutils import nodes
+from docutils.parsers.rst import Directive
+
+from contextlib import redirect_stdout
+from io import StringIO
+from pydoctor.driver import parse_args
+
+
+class HelpOutputDirective(Directive):
+    """
+    Directive that will generate the pydoctor help as block literal.
+    """
+    has_content = True
+
+    def run(self):
+
+        stream = StringIO()
+        try:
+            with redirect_stdout(stream):
+                parse_args(['--help'])
+        except SystemExit:
+            pass
+
+        text = ['pydoctor --help'] + stream.getvalue().splitlines()[1:]
+        return [nodes.literal_block(text='\n'.join(text))]
+
+
+def setup(app):
+    app.add_directive('help_output', HelpOutputDirective)
+
+    return {
+            'version': '0.1',
```

Comment:
Maybe use the pydoctor version instead?

Reply:
This is only to be use for internal / private pydoctor documentation... I would keep it simple and just use a static version.
I have no idea how this version is supposed to be used.

Reply:
If it shows up in the logging, having an actual version there might be useful. But if it's entirely invisible, it doesn't matter. Will Sphinx accept it if you remove this item altogether?

Reply:
It accepts without a version

The code is here https://github.com/sphinx-doc/sphinx/blob/4c1b8b8db6fa25afcab1a6a3634776009cfafddf/sphinx/extension.py#L25

The version is used together with `needs_extensions`

I will use  the same version as pydoctor.